src/operator/race.ts

```diff
@@ -23,8 +23,7 @@ export function race<T>(...observables: Array<Observable<T> | Array<Observable<T
     observables = <Array<Observable<T>>>observables[0];
   }
 
-  observables.unshift(this);
-  return raceStatic.apply(this, observables);
+  return this.lift.call(raceStatic<T>(this, ...observables));
```

Comment:
Shouldn't be. The instance method unwraps the argument if it's an Array of Observables, but I just looked and we don't have any test for this, which makes me nervous.